lib/fog/libvirt/models/compute/server.rb

```diff
@@ -351,14 +351,18 @@ def defaults
           }
         end
 
+        def valid_boot_media
+          %w[cdrom fd hd network]
+        end
+
         def default_boot_order
           %w[hd cdrom network]
         end
 
         def verify_boot_order order = []
           if order
             order.each do |b|
-              raise "invalid boot order, possible values are: hd, network and/or cdrom" unless default_boot_order.include?(b)
```

Comment:
Agreed. Same applies to the `valid_boot_media()` method I introduced, so I removed that one as well. Please have a look at the new commit I just pushed.